### DISCUSSION TITLE
1.8.2 cannot login admin, always ask for "Please log in"

### DISCUSSION THREAD

**Discussion by irvinyip** [YOURLS/YOURLS#3253] [Q&A]
Hi All,

I just deployed 1.8.2 release as a docker container, map on host 80 port (http://url.docker.local). **Everything works** fine if I change config.php to define( 'YOURLS_PRIVATE', false ); MySQL database records get in properly and shorten links also works.

However, if I change define( 'YOURLS_PRIVATE', true ); I can't login even login name and password is correct, no matter I disable the hashing to make sure the password match with config.php. It always show "Please log in".

/.htaccess is default:
```
# BEGIN YOURLS
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteBase /
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^.*$ /yourls-loader.php [L]
</IfModule>
# END YOURLS
```

I also tried enable debug and only see on footer:
```
Connected to database my_yourls on x.x.x.x 
SQL perform: SELECT option_name, option_value FROM yourls_options WHERE 1=1 (0.00036 s)
Check for new version: no
```
Browser developer mode also shows no error.

Any suggestion on troubleshooting is welcome.

Thanks.

**Comment by ozh** [comment]
Docker noob here, I'm rarely using it so there may be better stuff to do than my following advice.

What * I * would do in such a situation: edit `includes/functions-auth.php` in the Docker container and temporarily add various `var_dump()` here and there, starting with [`yourls_is_valid_user()`](https://github.com/YOURLS/YOURLS/blob/3828863675ce8257704e402ac4d9f7c8ee396168/includes/functions-auth.php#L24). The first thing I would investigate is whether the cookie is correctly created or not, which might explain the situation.

**Comment by gildesaant** [comment]
I'm having the same issue, have you managed to get it work?

**Comment by irvinyip** [comment]
Finally I protect the admin folder access with .htaccess with basic authentication, and my purpose is achieved.
